docs/testing.rst

```diff
@@ -68,6 +69,37 @@ The :ref:`Project` object for your project.
         assert 'MyContract' in project.compiled_contract_data
 
 
+How populus finds the project of the project fixture
+''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+If no other argument is provided, populus assumes that the tested project directory is the one where the tests run from.
```

Comment:
Simpler wording would be something like *"populus defaults to using the current working directory"*

Reply:
tests directory, since you can run py.test /path/to/project